src/textual/widget.py

```diff
@@ -806,7 +806,7 @@ def get_component_rich_style(self, *names: str, partial: bool = False) -> Style:
         """Get a *Rich* style for a component.
 
         Args:
-            name: Name of component.
+            names: Names of components.
```

Comment:
This whole docstring for `get_component_styles` might need updating to reflect recent changes?